### PR TITLE
Implement missing options(--with-wal-blocksize, --with-segsize, --with-pgport, --enable-tap-tests)

### LINKED ISSUE #15: --with-wal-blocksize / XLOG_BLCKSZ (easy)

### PR DESCRIPTION
**Pull request by nbyavuz**
Fixes #15, #16, #17 and #18

### CODE REVIEW

File: meson_options.txt [hunk 1]
```diff
@@ -21,6 +21,47 @@ option('extra_include_dirs', type : 'array', value: [],
 option('extra_lib_dirs', type : 'array', value: [],
   description: 'non-default directories to be searched for libs')
 
+option('enable-tap-tests', type : 'boolean', value : false,
```
Comment (anarazel) [inline meson_options.txt:24]: Let's have this default to on for now...

**Review by anarazel — COMMENTED** [review]
Unfortunately I just rebased the tree, so you need to do as well :(. But OTOH, I hope all tests should pass again, so there's that.

File: meson_options.txt (hunk 1 above)
Comment (nbyavuz) [reply meson_options.txt]: Done.

**Comment by nbyavuz** [comment]
> Unfortunately I just rebased the tree, so you need to do as well :(. But OTOH, I hope all tests should pass again, so there's that.

Done.

**Pull request merged**